06.bowling_object/game.rb

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require_relative 'frame'
+
+class Game
+  def initialize(shot_scores)
+    @frames = [Frame.new(0)]
+
+    shot_scores.gsub('X', '10').split(',').each do |shot_score|
+      record_shot(shot_score.to_i)
```

Comment:
record_shotの中で`@frames`に要素が追加されるというのはメソッド名から想像が付きにくいです。
たとえば、

```suggestion
      frame = record_shot(shot_score.to_i)
      @frames << frame if frame
```

みたいになっている方が「initializeで全部`@frames`をセットするんだな」ということが明確になってよいと思います。
（その上で`record_shot`というメソッドが適切かどうかを検討してみてください）

Reply:
`@frames`に対する操作が初期化処理内で行われていることが明確になるように、初期化処理の中で`@frames`に対して要素を追加するように変更しました。